scheduler/src/cook/compute_cluster.clj

```diff
@@ -324,6 +338,24 @@
                false)
     false))
 
+(defn config=?
+  "Returns true if we can consider current-config and
```

Comment:
nitpick, but mind not wrapping so hard. like have "Returns true if we can consider current-config and new-config to be equal." on one line

Reply:
Done